code/modules/admin/verbs/debug.dm

```diff
@@ -108,8 +104,8 @@ GLOBAL_LIST_EMPTY(AdminProcCallSpamPrevention)
 GLOBAL_PROTECT(AdminProcCallSpamPrevention)
 
 /proc/WrapAdminProcCall(datum/target, procname, list/arguments)
-	if(target && procname == "Del")
```

Comment:
Why this change ? Destroy is completely unrelated.

Reply:
Because `Destroy()` shouldn't be called by anything but `qdel()`.
It's reverted though.

Reply:
In code sure,  but this is debugging tool. Del is only here to prevent privilege escalation.

Reply:
Fair enough, changes done btw.